### PR TITLE
Feature/tao 7929/deny spellchecker

### PR DESCRIPTION
**Pull request by zagovorichev**
https://oat-sa.atlassian.net/browse/TAO-7929

The plugin that allows us to turn off the default spellcheck.

### CODE REVIEW

File: views/js/runner/plugins/content/preventSpellchecker.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+        /**
+         * Initialize the plugin, called during the test runner's initialization
+         */
+        init: function init() {
+            var config = _.defaults((this.getConfig() || {}), _defaults);
+
+            this.getTestRunner()
+                .after('renderitem', function () {
+                    $(config.selector).attr('spellcheck', 'off');
```
Comment (hutnikau) [inline views/js/runner/plugins/content/preventSpellchecker.js]: I thinks it's better to not replace this attribute globally but just in the elements which are part of item content:
```
var content = this.getTestRunner().getAreaBroker().getContentArea();
content.find(config.selector).attr('autocorrect', 'off');
```

File: views/js/runner/plugins/content/preventSpellchecker.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                    $(config.selector).attr('autocorrect', 'off');
```
Comment (hutnikau) [inline views/js/runner/plugins/content/preventSpellchecker.js]: According to documentation the `spellcheck` attribute has `true|false` values (but not `off`).
I would suggest to use the following set of attributes:
```
autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false"
```

Comment (zagovorichev) [reply views/js/runner/plugins/content/preventSpellchecker.js]: I don't think that we need to use autocapitalize="off" and autocomplete="off" here, it is not related to spellchecker directly

**Comment by zagovorichev** [comment]
@hutnikau Update

**Pull request merged**